common-practices-tools/contribution/contrib-first.md

```diff
@@ -42,6 +42,19 @@ See more of [CivicActions Drupal contributions](https://drupal.org/civicactions)
 
 <!--lint disable maximum-heading-length-->
 
+### How do we decide on a solution?
+
+The choice of where we create a solution follows this decision tree.
+
+```mermaid
+flowchart TD
+  A(Accomplish requirements with your existing packages) --> B(Leverage APIs from existing packages)
+  B --> C(Add another open-source package)
+  C --> D(Extend an existing open-source package with a contributed patch)
+  D --> E(Write & Contribute a new open-source package)
+  E --> F(Write custom code)
+```
```

Comment:
@swirtSJW looks like it rendered as code https://civicactions-handbook--1533.org.readthedocs.build/en/1533/common-practices-tools/contribution/contrib-first/ so we might need something like https://mkdocs-mermaid2.readthedocs.io/en/latest/

Reply:
Do you want to take a shot at adding it? See https://squidfunk.github.io/mkdocs-material/reference/diagrams/?h=mermaid for our theme

Reply:
Shot taken :)

Reply:
Hey, it worked!

![Screenshot of mermaid diagram on guidebook page](https://github.com/user-attachments/assets/aa8f4309-9eba-4f62-8e68-65e193791d69)